manual_user_install.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+# tested with a clean 64 bit installation of Raspbian-Lite
+
+function install_core (){
+    echo "Installing OVOS core"
+    echo
+    pip3 install -U pip
+
+    # install ovos-core - ovos-backend-client has conflicting dependencies
```

Comment:
what conflicts?

Reply:
Gets rid of the warning that pip needs updated.  Should this be removed? or does it really matter?  I would just assume leave it

Reply:
i want to know what the issue is so i can fix it :P a bug report would be better than this comment

Reply:
oh, i meant the comment "ovos-backend-client has conflicting dependencies" not the removal of `pip3 install -U pip`, that is fine to remove

imho if it doesnt make a difference, don't add it to keep variables down :)